src/components/button/index.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import React from 'react';
+
+import { IProps } from './types';
+
+/**
+ * @brief Button 컴포넌트
+ * @description label 필수, type defalut = 1
+ * @param label
+ * @param type 1(Primary), 2(PrimaryAlternate), 3(Secondary), 4(SecondaryAlternate), 5(Disabled), 6(DisabledAlternate)
+ * @param onClick
+ * @param className tailwind 요소
+ */
+
+const Button = ({ label, type = 1, onClick, className, ...props }: IProps) => {
+  const baseStyle = `${className} relative flex h-[48px] w-[182px] flex-col items-stretch justify-center rounded-lg border`;
+
+  // 각 버튼 타입에 따른 추가 스타일
+  const typeStyles = {
+    1: 'bg-primaries-primary border-primaries-primary p-6 text-text-20 hover:bg-primaries-hover active:bg-primaries-active',
```

Comment:
피그마처럼 disabled일 때도 색을 처리하면 좋을 것 같은데 어떠신가용?? ㅎㅎ
+) hover, active일 때 border도 따로 적용시켜 보았어요!

```suggestion
1: 'bg-primaries-primary p-6 text-text-20 border-primaries-primary hover:bg-primaries-hover hover:border-primaries-hover active:border-primaries-active active:bg-primaries-active disabled:bg-text-40 disabled:text-text-20 disabled:text-opacity-50 disabled:border-text-40',

```